pkg/controller/perconaservermongodb/ssl.go

```diff
@@ -226,15 +226,31 @@ func getShardingSans(cr *api.PerconaServerMongoDB) []string {
 		"*." + cr.Name + "-" + api.ConfigReplSetName + "." + cr.Namespace,
 		"*." + cr.Name + "-" + api.ConfigReplSetName + "." + cr.Namespace + "." + cr.Spec.ClusterServiceDNSSuffix,
 	}
+	if cr.Spec.MultiCluster.Enabled {
+		sans = append(sans, []string{
+			cr.Name + "-mongos" + "." + cr.Namespace + "." + cr.Spec.MultiCluster.DNSSuffix,
+			"*." + cr.Name + "-mongos" + "." + cr.Namespace + "." + cr.Spec.MultiCluster.DNSSuffix,
+			cr.Name + "-" + api.ConfigReplSetName + "." + cr.Namespace + "." + cr.Spec.MultiCluster.DNSSuffix,
+			"*." + cr.Name + "-" + api.ConfigReplSetName + "." + cr.Namespace + "." + cr.Spec.MultiCluster.DNSSuffix,
+		}...)
+	}
+	return sans
 }
 
 func getCertificateSans(cr *api.PerconaServerMongoDB, replset *api.ReplsetSpec) []string {
-	return []string{
+	sans := []string{
 		cr.Name + "-" + replset.Name,
 		cr.Name + "-" + replset.Name + "." + cr.Namespace,
 		cr.Name + "-" + replset.Name + "." + cr.Namespace + "." + cr.Spec.ClusterServiceDNSSuffix,
 		"*." + cr.Name + "-" + replset.Name,
 		"*." + cr.Name + "-" + replset.Name + "." + cr.Namespace,
 		"*." + cr.Name + "-" + replset.Name + "." + cr.Namespace + "." + cr.Spec.ClusterServiceDNSSuffix,
 	}
+	if cr.Spec.MultiCluster.Enabled {
+		sans = append(sans, []string{
+			cr.Name + "-" + replset.Name + "." + cr.Namespace + "." + cr.Spec.MultiCluster.DNSSuffix,
+			"*." + cr.Name + "-" + replset.Name + "." + cr.Namespace + "." + cr.Spec.MultiCluster.DNSSuffix,
+		}...)
+	}
+	return sans
```

Comment:
What happens if you enable MCS on an existing cluster? Does mongod pods require a restart?

Reply:
If I enable MCS on an already existing cluster, the certificates change, and mongod will restart

Reply:
is it really work like that? as I can see `reconsileSSL` function only check if SSL secret exists and don't recreate it

Reply:
@pooknull have you double checked?

Reply:
4ff8066c7e5a6a6db4b964e53a48be501ff30748